tests/datavolume_test.go

```diff
@@ -548,12 +467,6 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]DataVolume tests",
 					Type:   cdiv1.DataVolumeReady,
 					Status: v1.ConditionTrue,
 				},
-				boundCondition: &cdiv1.DataVolumeCondition{
```

Comment:
Why are we removing these? Because we don't know the exact name of the PVC and its in the comparison string?

Reply:
Because the name of the PVC can be only filled in runtime, the `boundCondition` section in each table entry was identical. I moved it to line 161 to remove repetition of the same block.